content/en/docs/reference/scheduling/config.md

```diff
@@ -289,6 +291,186 @@ only has one pending pods queue.
 {{% /tab %}}
 {{< /tabs >}}
 
+### Plugins that apply to multiple extension points {#multipoint}
```

Comment:
Wait, this section should be after `Multiple profiles` and before `Scheduler configuration migrations`